### ISSUE TITLE
Installation fails - invalid HTTPS response

### ISSUE THREAD

**Issue by DeltaLima** [hlavki/g-suite-identity-sync#6]
Hi there! 
I tried to install the g-suite-identity-sync but i fail at the same point. When i try to open up https://localhost:8181 then i get the response: 

Error code: <a id="errorCode" title="SSL_ERROR_RX_RECORD_TOO_LONG">SSL_ERROR_RX_RECORD_TOO_LONG</a> 

I followed the installation guide step by step. Got no errors.+

I want to use this script in a production environment in the company where i started few months ago. 
I'm bit confused what Karaf does in this thing ? And what is the "key magic"? I just need The Google Gsuite exported as LDAP - and this stable :) 

Do you recommend the script in it's actual state for production? We would like to support you and your work on this project :)

Thanks and best regards from Berlin!

Marcus

**Comment by hlavki** [comment]
Yeah, another mistake in documentation. This is change made in version 0.3.1. You have to change _Authorized Redirect URI_ to http://localhost:8181/cxf/oidc/rp/complete

See: [Create authorization credentials for Web Application](https://github.com/hlavki/g-suite-identity-sync/wiki/GSuite-configuration#create-authorization-credentials-for-web-application)

**Comment by DeltaLima** [comment]
Cool, thank you! :) I'll try this!
